src/parser/csql_grammar.y

```diff
@@ -24764,26 +24760,29 @@ dblink_expr
                         PT_NODE_PRINT_VALUE_TO_TEXT (this_parser, val);
 		   }
 
-                if( $1->node_type == PT_NAME )
+                if( $1 )
                 {
-                        ct->info.dblink_table.is_name = true;
-                        ct->info.dblink_table.conn = $1;       
+                        if( $1->node_type == PT_NAME )
```

Comment:
When processing the 'alter_stmt' rule, PT_NODE is first created, and the necessary processing is performed in the sub-rule using the parser_save_alter_node() and parser_get_alter_node() functions.

Similarly, how about creating PT_NODE *ct first, and then using the parser_save_ct_node() and parser_get_ct_node() functions to perform the necessary processing according to PT_NAME and PT_VALUE in dblink_conn, a sub-rule of dblink_expr?

Reply:
As you said, it may be possible to modify it, but for now, we will maintain the status quo.